docs/references/expo/offline-support.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Enable offline support in your Expo app
+description: Learn how to enable offline support in your Expo app with Clerk.
+---
+
+> [!WARNING]
+> **This is an experimental API.**
+>
+> The `__experimental_resourceCache` property introduced in this guide is an experimental feature. It is subject to change in future updates, so use it cautiously in production environments. Ensure thorough testing and stay informed about updates in the package.
```

Comment:
Can we share where/how to stay informed here?

Reply:
Hmm, should we point them to the changelog of our Expo package? 🤔

Reply:
If that's the spot where updates will be published, then I think that's the best choice.

Reply:
Yes, this is the only spot I believe

Reply:
Added the link to the changelog!